### ISSUE TITLE
When we add a new reminder in the middle of the year, they aren't added to exisiting schools

### ISSUE THREAD

**Issue by michiQ** [yadaguru/yadaguru-app#120]
So I found this one out when making the change for test dates.  I added a new reminder for testing registration and it didn't show up on the schools I already set up.  It *did* show up on a new school that I created.  I think that it would probably be pretty rare that we would do this. I think its just something to do if we have time and good to know for the future.

**Comment by bobderrico80** [comment]
Yeah this could become an issue. When a user adds a new school, the current set of base reminders are generated for the school for the user. At a minimum, this is something we will have to think about when adding or deleting reminders (updating is fine, as the entire reminder is not copied, but is associated with a foreign key). We should probably add logic to check and update user's reminders when adding or deleting an existing reminder.
